<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaAlterarCliente</class>
 <widget class="QDialog" name="TelaAlterarCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>E-MAIL:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditEmail"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonInserirImagem">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>INSEIR IMAGEM</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CPF:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditCpf"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CNH:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditCnh"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DDD:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditDDD"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DDI:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditDDI"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonCancelar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCELAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImagem">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NÃšMERO:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditNumero"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NOME:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditNome"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonAlterar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ALTERAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tipo de Relacionamento:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTipoRelacionamento">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Normal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Premium</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VIP</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditNome</tabstop>
  <tabstop>lineEditCpf</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>lineEditCnh</tabstop>
  <tabstop>lineEditDDI</tabstop>
  <tabstop>lineEditDDD</tabstop>
  <tabstop>lineEditNumero</tabstop>
  <tabstop>pushButtonInserirImagem</tabstop>
  <tabstop>comboBoxTipoRelacionamento</tabstop>
  <tabstop>pushButtonAlterar</tabstop>
  <tabstop>pushButtonCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
